R/as.grambank.wide.R

```diff
@@ -0,0 +1,20 @@
+#' Extract a wide dataframe from a grambank value table
```

Comment:
I think that a function like this could live in rcldf instead. I get questions more often than I'd expect on how to make ValueTables wide, and this function and the one I wrote before is super simple and works on any cldf ValueTable. I think people just need to be hand-held through reshape::dcast/tidyr::pivot_wider/tidyr::spread sometimes

Reply:
A function like that in `rcldf` would need to identify relevant columns by propertyUrl in the associated metadata, though, not by column name.

Reply:
yes, the implementation in rcldf would need to be more complicated -- and for some datasets could be explosively bad (e.g. dplace has 2500 variables, do people really want a 2500 col dataframe?) I think we should keep this here for now for grambank purposes and think about implementing something in rcldf later.